x-pack/plugin/core/src/main/resources/ecs-core-mappings.json

```diff
@@ -0,0 +1,318 @@
+{
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "ecs_timestamp": {
+          "mapping": {
+            "ignore_malformed": false,
+            "type": "date"
+          },
+          "match": "@timestamp"
+        }
+      },
+      {
+        "message_match_only_text": {
+          "mapping": {
+            "type": "match_only_text"
+          },
+          "match": "message"
+        }
+      },
+      {
+        "ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "match": "ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "_ip_to_ip": {
+          "mapping": {
+            "type": "ip"
+          },
+          "path_match": "*_ip",
+          "match_mapping_type": "string"
+        }
+      },
+      {
+        "wildcard": {
+          "mapping": {
+            "type": "wildcard"
+          },
+          "path_match": [
+            "*.io.text",
+            "registry.data.strings",
+            "*url.path"
+          ]
+        }
+      },
+      {
+        "command_line_to_multifield": {
+          "mapping": {
+            "fields": {
+              "text": {
+                "type": "match_only_text"
+              }
+            },
+            "type": "wildcard"
+          },
+          "match": "command_line"
+        }
+      },
+      {
+        "error_stack_trace_to_multifield": {
+          "mapping": {
+            "fields": {
+              "text": {
+                "type": "match_only_text"
+              }
+            },
+            "type": "wildcard"
+          },
+          "match": "stack_trace"
+        }
+      },
+      {
+        "http_content_to_multifield": {
+          "mapping": {
+            "fields": {
+              "text": {
+                "type": "match_only_text"
+              }
+            },
+            "type": "wildcard"
+          },
+          "path_match": "*.body.content"
```

Comment:
Can we simplify to just say `*.content` is wildcard?

Reply:
I'd say `content` may generally sound like a full-text-search field, but in [this case](https://www.elastic.co/guide/en/ecs/8.7/ecs-http.html#field-http-request-body-content) it is _**also**_ mapped to `wildcard`.